dockerfiles/start.sh

```diff
@@ -2,13 +2,9 @@
 
 if [[ "${DCAP_LIBRARY}" == "intel" ]]
 then
-    # rename the library installed by az-dcap-client
-    mv /usr/lib/libdcap_quoteprov.so /usr/lib/libdcap_quoteprov.so.azure
-    # create a link to the intel quote provider library
-    ln -s /usr/lib/x86_64-linux-gnu/dcap/libdcap_quoteprov.so.intel /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so
-    ln -s /usr/lib/x86_64-linux-gnu/dcap/libdcap_quoteprov.so.intel /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so.1
+    apt-get install -qq libsgx-dcap-default-qpl
```

Comment:
```suggestion
    apt-get install -qq libsgx-dcap-default-qpl > /dev/null
```

There is still *some* output even with `-qq`

Reply:
Do you think the output is undesirable? I don't have an opinion on this. Will follow your suggestion if you confirm.

Reply:
I think its a bit unsightly to get the "Reading package database" messages every time you start the container.
We don't lose anything by hiding it, but logs look a bit messy if we do.